src/python/deepgnn/pytorch/common/args.py

```diff
@@ -10,7 +10,7 @@
 def init_model_common_args(parser: argparse.ArgumentParser):
     """Model configuration."""
     group = parser.add_argument_group("Model Common Parameters")
-    group.add_argument("--feature_type", default="float", type=str, choices=["float", "uint64", "binary"], help="Feature type.")
+    group.add_argument("--dtype", default="float", type=str, choices=["float", "uint64", "binary"], help="Feature type.")
```

Comment:
please, keep argument name `feature_type`, it is more obvious than an obscure dtype reference.